src/lib/seam/components/ClimateSettingScheduleDetails/ClimateSettingScheduleDetails.tsx

```diff
@@ -75,55 +80,82 @@ export function ClimateSettingScheduleDetails({
+                  <ArrowRightIcon />
+                  {formatDateTime(climateSettingSchedule.schedule_ends_at)}
+                </span>
+              </DetailRow>
+              <DetailRow label={t.climateSetting}>
+                <ClimateSettingStatus
+                  climateSetting={climateSettingSchedule}
+                  iconPlacement='right'
+                />
+              </DetailRow>
+              <DetailRow label={t.allowManualOverride}>
+                <span className='seam-climate-setting-details-value'>
+                  <Switch
+                    checked={isManualOverrideAllowed}
+                    onChange={(checked) => {
+                      mutate({
+                        climate_setting_schedule_id:
+                          climateSettingSchedule.climate_setting_schedule_id,
+                        manual_override_allowed: checked,
+                      })
+                    }}
+                  />
+                </span>
+              </DetailRow>
+            </DetailSection>
+            <DetailSection>
+              <DetailRow label={t.creationDate}>
+                <div className='seam-creation-date'>
+                  {formatDateTime(climateSettingSchedule.created_at)}
+                </div>
+              </DetailRow>
+            </DetailSection>
+          </DetailSectionGroup>
         </div>
-        <DetailSectionGroup>
-          <DetailSection>
-            <DetailRow label={t.startEndTime}>
-              <span className='seam-climate-setting-details-value seam-climate-setting-details-schedule-range'>
-                {formatDateTime(climateSettingSchedule.schedule_starts_at)}
-                <ArrowRightIcon />
-                {formatDateTime(climateSettingSchedule.schedule_ends_at)}
-              </span>
-            </DetailRow>
-            <DetailRow label={t.climateSetting}>
-              <ClimateSettingStatus
-                climateSetting={climateSettingSchedule}
-                iconPlacement='right'
-              />
-            </DetailRow>
-            <DetailRow label={t.allowManualOverride}>
-              <span className='seam-climate-setting-details-value'>
-                {isManualOverrideAllowed ? t.on : t.off}
-              </span>
-            </DetailRow>
-          </DetailSection>
-          <DetailSection>
-            <DetailRow label={t.creationDate}>
-              <div className='seam-creation-date'>
-                {formatDateTime(climateSettingSchedule.created_at)}
-              </div>
-            </DetailRow>
-          </DetailSection>
-        </DetailSectionGroup>
       </div>
-    </div>
+      <Snackbar
+        message={t.manualOverrideSuccess}
+        variant='success'
+        visible={isSuccess}
+        automaticVisibility
+      />
+
+      <Snackbar
+        message={t.manualOverrideError}
+        variant='error'
+        visible={isError}
+        automaticVisibility
+      />
+    </>
```

Comment:
Huge diff here because of nesting inside a fragment, but it really only:

- adding manual override switch

```tsx
    <Switch
                    checked={isManualOverrideAllowed}
                    onChange={(checked) => {
                      mutate({
                        climate_setting_schedule_id:
                          climateSettingSchedule.climate_setting_schedule_id,
                        manual_override_allowed: checked,
                      })
                    }}
                  />
```

and the snackbars:

```tsx
    <Snackbar
        message={t.manualOverrideSuccess}
        variant='success'
        visible={isSuccess}
        automaticVisibility
      />

      <Snackbar
        message={t.manualOverrideError}
        variant='error'
        visible={isError}
        automaticVisibility
      />
```